src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/BrowseOAFeatureService/BrowseOAFeatureService.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.WPF.Samples.BrowseOAFeatureService.BrowseOAFeatureService"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="*" />
+                    <RowDefinition Height="Auto" />
+                    <RowDefinition Height="Auto" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
+                </Grid.ColumnDefinitions>
+                <TextBlock
+                    Grid.Row="0" Grid.ColumnSpan="2"
+                    Margin="5"
+                    FontWeight="Bold" Foreground="Black" Text="Load the service, then select a layer for display." TextAlignment="Center" />
+                <TextBox x:Name="ServiceTextBox"
+                    Grid.Row="1" Grid.Column="0"
+                    Margin="0,0,5,5" />
+                <Button x:Name="LoadServiceButton"
```

Comment:
This could benefit from a run through XAMLStyler if not run already. I mostly just want to see attributes on newlines aligned with the first attribute on the element.